libs/ledger-live-common/src/wallet-api/react.ts

```diff
@@ -218,27 +220,32 @@ function usePermission(manifest: AppManifest): Permission {
 
 function useTransport(
   postMessage: (message: string) => void | undefined
-): Transport | undefined {
-  const transportRef = useRef<Transport>();
-
-  const transport = useMemo<Transport | undefined>(() => {
-    if (!postMessage) return;
-
+): Transport {
+  return useMemo(() => {
     return {
       onMessage: undefined,
       send: (message) => {
         postMessage(message);
       },
     };
   }, [postMessage]);
+}
 
-  useEffect(() => {
-    if (!transport) return;
-
-    transportRef.current = transport;
-  }, [transport]);
+export function useConfig({
+  tracking,
+  wallet,
+}: Pick<ServerConfig, "tracking" | "wallet">): ServerConfig {
+  const userId = useEnv("USER_ID");
 
-  return transportRef.current;
+  return useMemo(
+    () => ({
+      appId: "ledger-live",
```

Comment:
🤔 Not sure what to put. If it's from manifest, it have to be set after instantiating `WalletAPIServer`.